src/System.Runtime.Environment/src/RuntimeInfo.Linux.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Runtime.InteropServices
+{
+    public static class RuntimeInfo
+    {
+        public static bool IsOSPlatform(OSName osName)
+        {
+            if (osName.Equals(OSName.Linux))
+                return true;
```

Comment:
This can also nullref, I'd revert the comparison as we know that OSName.Linux._osName will never be null